app/controllers/concerns/offender_sar_cases_params.rb

```diff
@@ -38,6 +38,7 @@ def create_offender_sar_params
       :request_method,
       :requester_reference,
       :current_state,
+      :rejected,
```

Comment:
Where is this param used?

Reply:
This rejected param is being used in _case_details.html.slim

Reply:
The params in this file are used to allow updating of attributes via a form. It's a security measure called Strong Parameters - https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters.

In _case_details.html.slim you are checking the value of an attribute which is an unrelated action.